phpunit.xml

```diff
@@ -5,6 +5,11 @@
             <directory>Tests</directory>
         </testsuite>
     </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">src</directory>
+        </whitelist>
+    </filter>
```

Comment:
To evaluate the code coverage, it should add the filter white list to achieve this.